src/drivers/esp32/src/esp_slip.c

```diff
@@ -72,11 +72,11 @@ static void sendSlipPacket(uint32_t size, uint8_t *data, coms_sendbuffer_t sendB
 
   for (i = 0; i < size; i++)
   {
-    if ((data[i] == 0xC0 && i != 0 && i != size - 1) || (data[i] == 0xDB && i != 0 && i != size - 1))
+    if ((data[i] == SLIP_START_STOP_BYTE && i != 0 && i != size - 1) || (data[i] == 0xDB && i != 0 && i != size - 1))
     {
       for (int j = 0; j < 2; j++)
       {
-        j == 0 ? (dmaSendBuffer[dmaSendSize] = 0xDB) : data[i] == 0xC0 ? (dmaSendBuffer[dmaSendSize] = 0xDC)
+        j == 0 ? (dmaSendBuffer[dmaSendSize] = 0xDB) : data[i] == SLIP_START_STOP_BYTE ? (dmaSendBuffer[dmaSendSize] = 0xDC)
```

Comment:
I really would prefer if-statements here, ternary are a hell to parse while reading code. And code is read 1000x more often than it is written, at least. Always focus on readability when in doubt.